mosaic-site/src/layouts/DetailTechnical/DetailTechnical.module.css

```diff
@@ -16,6 +16,36 @@
   --border-width-medium: 3px;
 }
 
+.primarySidebar :global(.ps-menu-button) {
```

Comment:
just curious, the `ps-menu-button` on hover seems to have the text touching the left edge, seems like it is because it is not finding  `var(--space-horizontal-x4)` and there are other paddings declared but being overriden, might be worth checking on that?

Reply:
Would you be able to post a screenshot please I think that may have already been fixed